.github/workflows/dev_pr/link.js

```diff
@@ -51,11 +51,30 @@ async function commentJIRAURL(github, context, pullRequestNumber, jiraID) {
   });
 }
 
+async function commentGitHubURL(github, context, pullRequestNumber, issueID) {
```

Comment:
Can we start adding docstrings to those functions? It's not nice to have a growing set of uncommented workflow helpers.

Reply:
Thanks you are correct, I've added a bunch of docstrings to new and existing methods.